docs/tutorial/08-create-libs.md

```diff
@@ -15,8 +15,8 @@ To illustrate how useful libraries can be, create a library of Angular component
 ```treeview
 myorg/
 ├── apps/
-│   ├── frontend/
-│   ├── frontend-e2e/
+│   ├── todos/
```

Comment:
Could you also make the same fix in 07-share-code?

Reply:
@vsavkin, done. The changes were applied.